voltdb/README.md

```diff
@@ -78,6 +78,32 @@ If [TLS/SSL][5] is enabled on the client HTTP port:
 
 3. [Restart the Agent][4].
 
+#### Log Collection
+
+_Available for Agent versions >6.0_
```

Comment:
```suggestion
```

I don't think we need this note now

Reply:
Interesting, thanks. I mostly based this off the new docs spec so it means we may have some changes required there as well. #7891